x-pack/plugins/security/server/authentication/providers/oidc.ts

```diff
@@ -289,11 +293,13 @@ export class OIDCAuthenticationProvider extends BaseAuthenticationProvider {
     try {
       // This operation should be performed on behalf of the user with a privilege that normal
       // user usually doesn't have `cluster:admin/xpack/security/oidc/prepare`.
-      const {
-        state,
-        nonce,
-        redirect,
-      } = await this.options.client.callAsInternalUser('shield.oidcPrepare', { body: params });
+      const { state, nonce, redirect } = (
+        await this.options.client.asInternalUser.transport.request({
+          method: 'POST',
+          path: '/_security/oidc/prepare',
+          body: params,
+        })
+      ).body as any;
```

Comment:
![image](https://user-images.githubusercontent.com/3493255/105372982-2d87c900-5bd4-11eb-9bcc-0010262ffbbf.png)


I kid, I think this is tolerable for now given our usage

Reply:
:rofl: Yeah, that's exactly how I feel about it :see_no_evil: